Source/WebGPU/WebGPU/RenderPipeline.mm

```diff
@@ -500,7 +500,7 @@ static WGPUTextureViewDimension convertViewDimension(WGSL::TextureViewDimension
                 };
                 entries.append(bufferEntry);
 
-                bufferEntry.binding = newEntry.binding + 1;
+                bufferEntry.binding = newEntry.binding + 2;
```

Comment:
Is it worth adding a comment with the layout?

Reply:
Maybe but I thought it was clear since I add `+ 1` on line 493 then this needs to be `+ 2` since it is the next element. Or I could have done `++newEntry.binding` like on like 489